ocrd/ocrd/decorators/__init__.py

```diff
@@ -46,6 +52,40 @@ def ocrd_cli_wrap_processor(
             list_resources=list_resources
         )
         sys.exit()
+    if server_ip or server_port:
+        # IP provided without port
+        if server_ip and not server_port:
+            raise click.UsageError('--server-port is missing.')
+
+        # Port is provided without IP
+        if server_port and not server_ip:
+            raise click.UsageError('--server-ip is missing.')
+
+        # IP and port but without database
+        if server_ip and server_port and not mongo_url:
+            raise click.UsageError('--mongo-url is missing.')
+
+        # Proceed when both IP and port are provided
+        initLogging()
+
+        # Init a processor instance to get access to its information
+        processor = processorClass(workspace=None)
```

Comment:
This will not always work. The existing `Processor` constructor runs a `ParameterValidator`, which (depending on the ocrd-tool.json of the processor, notably if there are `required` params without `default`) can fail on `None`.